core/utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+
+from pure_pagination import EmptyPage
+from pure_pagination import Paginator
+from django.shortcuts import render
+
+
+def create_child_instance_from_parent(child_cls, parent_instance):
```

Comment:
This can't be fixed later on? I don't have a big enough picture in head to guess it :/